### ISSUE TITLE
v6.7.0 breaks binary_sensor config: HTML Element Attribute required(?)

### ISSUE THREAD

**Issue by hockey6611** [danieldotnl/ha-multiscrape#209]
## Version of the custom_component
6.7.0

## Configuration

```yaml
multiscrape:
  - resource: https://www.[exampleairport].com
    scan_interval: 3600
    sensor:
      - unique_id: tsa_precheck
        name: TSA Pre-Check Wait Time
        select: "div.pre-check > span.wait-number"
        unit_of_measurement: "min"
        icon: mdi:airplane-clock
        on_error:
          value: none
    binary_sensor:
      - unique_id: garage_parking_limited
        name: Parking Garage Limited Spaces
        select: 'div > div:nth-child(2) > div.parking-number > span.all'
        value_template: >
          {% if value == 'Limited' %}
            on
          {% else %}
            off
          {% endif %}
        device_class: problem
        icon: mdi:plane-car
        on_error:
          value: none
    button:
      - unique_id: garage_parking_button
        name: Parking Garage Open Spaces Refresh
```
### HTML Element:

```html
<span class="all">Limited</span>
```

## Describe the bug

Upon upgrade from 6.6.1 to 6.7.0 breaks binary_sensor config breaks and becomes unavailable in home assistant. Other sensors and buttons on same resource continue to work. reverting back to 6.6.1 resolves the issue. The error message in the log seems to be looking for the element attribute, however, I am using an attribute, only the element itself.

## Debug log

```text
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 455, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 731, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 844, in add_to_platform_finish
    await self.async_added_to_hass()
  File "/config/custom_components/multiscrape/entity.py", line 88, in async_added_to_hass
    self._update_sensor()
  File "/config/custom_components/multiscrape/binary_sensor.py", line 196, in _update_sensor
    self._set_icon(self._attr_native_value)
AttributeError: 'MultiscrapeBinarySensor' object has no attribute '_attr_native_value'

```

**Comment by danieldotnl** [comment]
Sorry, this took a while but solved now in v6.7.2

**Comment by hockey6611** [comment]
Thank you! updating to v6.7.2 solved the issue I was seeing. No change to my config, and the binary sensor is working correctly again.
